src/modules/iasql_functions/sql/after_install.sql

```diff
@@ -114,7 +114,13 @@ begin
     -- If a transaction occurred less than 2 min ago we skip
     IF _change_type != 'OPEN_TRANSACTION' AND (_ts IS NULL OR _ts < _almost_2_min_interval) THEN
       PERFORM iasql_begin();
-      PERFORM iasql_commit();
+      BEGIN
+        PERFORM iasql_commit();
+      EXCEPTION
```

Comment:
should we register the error as another log?

Reply:
the TS function for iasql_commit would be already catching errors and inserting the error log once this is landed https://github.com/iasql/iasql-engine/pull/1663